lib/enumerable/all.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Enumerable
+  def my_all?
+    raise 'Please provide a block' unless block_given?
+
```

Comment:
- [ ] my_all? when no block or argument is given returns true when none of the collection members are false or nil
- [ ] my_all? when a class is passed as an argument returns true if all of the collection is a member of such class
- [ ] my_all? when a Regex is passed as an argument returns true if all of the collection matches the Regex
- [ ] my_all? when a pattern other than Regex or a Class is given returns true if all of the collection matches the pattern

Reply:
>     * [ ]  my_all? when no block or argument is given returns true when none of the collection members are false or nil
```ruby
if block_given?
      my_each { |i| return false unless yield(i) }
    elsif arg.nil?
      my_each { |i| return false unless i }
```
Solved!

Reply:
Solved